src/tss2-tcti/tctildr-dl.c

```diff
@@ -113,7 +113,7 @@ handle_from_name(const char *file,
     }
     *handle = dlopen(file_xfrm, RTLD_NOW);
     if (*handle == NULL) {
-        LOG_ERROR("Failed to load TCTI for name \"%s\": %s", file, dlerror());
+        LOG_WARNING("Failed to load TCTI for name \"%s\": %s", file, dlerror());
```

Comment:
Can we downgrade this even further? Sucks that their is no way to get something like errno to check for ENOENT....

Reply:
Done, it is now `LOG_DEBUG` like the other `dlerror()` messages in this function and therefore hidden from the user by default.